added Lytx company. Updated README.md and company-profiles with compa…

ruikawahara opened the pull request:
…ny info

📝Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]`

ℹ️Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

- [ ] This PR contains housekeeping only (URL edits, copy changes etc) - if check, delete other lines that don't apply.
- [ ] I am an employee of the company mentioned and confirm all included details are correct.
- [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [x] The company directly hires employees. No bootcamps / freelance sites / etc
- [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [x] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - This is __required__ for new additions.
- [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [x] __Region__ details the geographic regions in which this company's employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This Pull Request should adhere to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

auto-nylen commented:
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

ruikawahara commented:
Added "Lytx" as company available for hired. 
Information that I got is based on recruiter that works on Lytx. 

Added company info on correct location of README.md.
Also added corresponding company profile.

auto-nylen commented:
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

auto-nylen commented:
:robot: :warning:

_Sorry, this bot is currently having issues. It's probably @nylen's fault, not yours. We'll look into it._

:robot: :warning:

dougaitken commented:
Build deploys were paused. Merging.
